### PR TITLE
Feat: Add Spellcheck GitHub Action

### PR DESCRIPTION
**Pull request by VijaySharma815**
GitHub Spellcheck Action: https://github.com/marketplace/actions/github-spellcheck-action

### CODE REVIEW

File: .wordlist.txt [hunk 1]
```diff
@@ -0,0 +1,349 @@
...
+wl
+yhghud
+ysysnfxmqzu
+GsisFk
+XZlILWk
+aPJsrovvUEDrE
+ai's
+oaIug
+pJDVFS
+oposite
```
Comment (5A11) [inline .wordlist.txt:349]: Where are these from?

Comment (5A11) [reply .wordlist.txt:349]: Some of these don't seem correct, `XZlILWk`, `ulf`, `qhucdjs`, etc

**Comment by 5A11** [comment]
@VijaySharma815 Please fix merge conflict

File: .wordlist.txt (hunk 1 above)
Comment (VijaySharma815) [reply .wordlist.txt:271]: I added some more configurations and update the wordlist also.

**Review by 5A11 — APPROVED** [review]
lgtm

**Pull request merged**